docs/concepts/alias.md

```diff
@@ -0,0 +1,36 @@
+## Alias Precedence
```

Comment:
As it is `alias` concept page, don't we need to add a description about what is an alias?